src/webgl/p5.RendererGL.Immediate.js

```diff
@@ -205,6 +205,12 @@ p5.RendererGL.prototype.endShape = function(
   this._processVertices(...arguments);
   this.isProcessingVertices = false;
 
+  // Check the number of vertices and if equal to 3
+  // Change the shapemode to traingle
+  if (this.immediateMode.geometry.vertices.length === 3) {
```

Comment:
I believe this works as it is, but just to be a bit more robust for future changes, maybe we can add `&& this.immediateMode.shapeMode === constants.TESS` here so that we don't accidentally switch when the shape mode is already changed to something other than the default?